test/kafka-ingest-open-loop/mzcompose.py

```diff
@@ -236,13 +196,10 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
 
 
 def workflow_smoke_test(c: Composition) -> None:
-    for arg in ["--upsert", "--enable-persistence"]:
-        c.workflow(
-            "default",
-            "--num-seconds=15",
-            "--records-per-second=1000",
-            arg,
-        )
-        c.kill("materialized")
-        c.rm("materialized", "testdrive", "kafka", destroy_volumes=True)
-        c.rm_volumes("mzdata", "pgdata")
+    c.workflow(
+        "default",
+        "--num-seconds=15",
+        "--records-per-second=1000",
+    )
+    c.rm("materialized", "testdrive", "kafka", destroy_volumes=True)
```

Comment:
did we intentionally lose the `c.kill("materialized")` that was here before?

Reply:
unintentional. fixed!